src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -319,4 +320,35 @@ public static boolean isBooleanLiteralType(final int tokenType) {
         return isTrue || isFalse;
     }
 
+    /**
```

Comment:
Why are these methods in this class? The Javadoc says:
```java
/**
 * Contains utility methods for tokens.
 *
 */
public final class TokenUtil {
```

Reply:
Shouldn't be there, moved it.